R/plot.R

```diff
@@ -2,175 +2,298 @@
-  return(forest.df(freq$net1, modelranfix, freq$lstx, ref, min, max))
+  return(forest.df(netresult = freq$net1, model = modelranfix, lstx = freq$lstx, ref = ref,
+                   min = min, max = max
+                   )
+         )
 }
 
-# 2a. Creates text displayed under forest plots 
+
+
+#' 2a. Creates text displayed under forest plots
+#' 
+#' @param freq List of NMA results created by freq_wrap().
+#' @param outcome_measure "MD", "SMD", "OR", "RR", or "RD".
+#' @param modelranfix "fixed" or "random".
+#' @return Text created by tau.df().
 texttau <- function(freq, outcome_measure, modelranfix){      
-  tau <- round(freq$net1$tau,2)
-  return(tau.df(tau, freq$net1$k, freq$net1$n, modelranfix, outcome_measure))
+  tau <- round(freq$net1$tau, 2)
+  return(tau.df(tau = tau, k = freq$net1$k, n = freq$net1$n, model = modelranfix, outcome = outcome_measure))
 }
 
+
+
+#' Create text to say outcomes are compared to the reference treatment.
+#'
+#' @param ref Reference treatment.
+#' @return Text as above.
 make_refText = function(ref) {
-  y <- paste("All outcomes are versus the reference treatment:", ref)
-  return(y)
+  return(paste("All outcomes are versus the reference treatment:", ref))
 }
 
-# 2b Treatment comparison and rank table
+
+
+#' 2b Treatment comparison and rank table
+#' 
+#' @param freq List of NMA results created by freq_wrap().
+#' @param modelranfix "fixed" or "random".
+#' @param rankopts "good" or "bad", referring to smaller outcome values.
+#' @return Ranking table created by netmeta::netleague().
 make_netrank <- function(freq, modelranfix, rankopts) {
-  league <- netleague(freq$net1, 
-                      comb.random=(modelranfix=="random"), comb.fixed = (modelranfix=="fixed"), 
-                      digits =2, seq= netrank(freq$net1, small = rankopts))
-  if (modelranfix=="random"){
-    leaguedf<- as.data.frame(league$random)
-  }
-  else {
-    leaguedf<- as.data.frame(league$fixed)
+  league <- netmeta::netleague(freq$net1, comb.random = (modelranfix == "random"),
+                               comb.fixed = (modelranfix == "fixed"), digits = 2,
+                               seq = netmeta::netrank(freq$net1, small = rankopts)
+                               )
+  if (modelranfix == "random") {
+    leaguedf <- as.data.frame(league$random)
+  } else if (modelranfix == "random") {
+    leaguedf <- as.data.frame(league$fixed)
+  } else {
+    stop("modelranfix must be 'fixed' or 'random'") 
   }
   return(leaguedf)
 }
 
-# 2c Inconsistency
 
+
+#' 2c Inconsistency
+#'
+#' @param freq List of NMA results created by freq_wrap().
+#' @param modelranfix "fixed" or "random".
+#' @return Data frame of inconsistency data created by netsplitresult.df().
 make_Incon <- function(freq, modelranfix) {
-  incona <- netsplit(freq$net1)
+  incona <- netmeta::netsplit(freq$net1)
   return(netsplitresult.df(incona, modelranfix))
 }
 
-# 3a Forest plot 
+
+
+#' 3a Forest plot 
```

Comment:
What's the difference between this function and the one on line 85? The descriptions should make it clear which one should be used in which circumstances.